flink-runtime/src/main/java/org/apache/flink/runtime/state/AbstractStateBackend.java

```diff
@@ -95,7 +95,7 @@ public OperatorStateBackend createOperatorStateBackend(
 			Environment env,
 			String operatorIdentifier) throws Exception {
 
-		return new DefaultOperatorStateBackend(env.getUserClassLoader());
+		return new DefaultOperatorStateBackend(env.getUserClassLoader(), env.getExecutionConfig());
```

Comment:
`ExecutionConfig` should be obtained from `StreamTask`. That is , you should add a parameter typed `ExecutionConfig` in this method and modify the implementation in `StreamTask#createOperatorStateBackend()`.